### ISSUE TITLE
"add"/"subtract" commands for setting player config values

### ISSUE THREAD

**Issue by thexaero** [thexaero/open-parties-and-claims#239]

**Comment by FinnSetchell** [comment]
agreed, this is essential for any server in order to allow players to add to their bonus chunk claims without resetting it to a new value each time.

**Comment by quintaneous** [comment]
Is this feature planned on being implemented?

**Comment by MadSciSlack** [comment]
I also need this badly. The ability to reward chunks does SO much for server and modpack developers.  There are almost no options for claim management tying into currency system or even quest systems.  I've got 350 quests, and can't use real estate as a reward on a chunks controlled server...  Please help!

**Comment by f-eg** [comment]
pls i need

**Comment by coreyderg** [comment]
I need this as well, I run a dedicated server with a modpack that has this mod, and I am currently unable to reward my players with more bonus chunks for paying certain items...

**Comment by Mapter322** [comment]
Hi! I ran into the same issue, and after finding out it wasn’t possible, I created a mod that adds items to increase Bonus Claims.

Look for **OPAC Bonus Claims** on Modrinth or CurseForge.
